portfolio/src/main/webapp/Home.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
The indentation/spacing here is quite a bit different than the other HTML files. Can you please reformat this?